src/test/java/org/jenkinsci/plugins/badge/actions/EmbeddableBadgeConfigsActionTest.java

```diff
@@ -0,0 +1,53 @@
+package org.jenkinsci.plugins.badge.actions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+
+import hudson.model.Job;
+import hudson.model.Run;
```

Comment:
Don't need these imports since the mock objects are not needed for the tests as written
```suggestion
```